### PR TITLE
profile.d: copy and use inputrc from localhost

### PR DESCRIPTION
**Pull request by tamsky**
## what

adds a small profile.d script that copies the user's homedir `.inputrc` if it exists at geodesic container startup.

## why

Readline keybindings are typically stored in a user's `~/.intputrc`.
This script allows those keybindings to smoothly flow into the geodesic env.

### CODE REVIEW

File: rootfs/etc/profile.d/inputrc.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# If we can, and do not have one, copy inputrc from localhost.
+[ -f ~/.inputrc ] && [ -f /localhost/.inputrc ] || {
```
Comment (osterman) [inline rootfs/etc/profile.d/inputrc.sh]: Don't you mean this?

```suggestion
[ ! -f ~/.inputrc ] && [ -f /localhost/.inputrc ] || {
```

File: rootfs/etc/profile.d/inputrc.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	cp /localhost/.inputrc ~/.inputrc &&
```
Comment (osterman) [inline rootfs/etc/profile.d/inputrc.sh]: Why the copy and not just bind?

Comment (osterman) [reply rootfs/etc/profile.d/inputrc.sh]: Any thoughts on symlinking instead?

Comment (tamsky) [reply rootfs/etc/profile.d/inputrc.sh]: I've implemented your "just bind" suggestion in the next commit.

File: rootfs/etc/profile.d/inputrc.sh (hunk 1 above)
Comment (tamsky) [reply rootfs/etc/profile.d/inputrc.sh]: Yes.  Thanks!

File: rootfs/etc/profile.d/inputrc.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# If we do not have one of our own, source inputrc from localhost.
+[ ! -f ~/.inputrc ] && [ -f /localhost/.inputrc ] &&
```
Comment (osterman) [inline rootfs/etc/profile.d/inputrc.sh]: Prefer to use `if [ ... ]; then` syntax than long chaining commands.

Comment (tamsky) [reply rootfs/etc/profile.d/inputrc.sh]: Updated.

**Pull request merged**